.gitignore

```diff
@@ -8,3 +8,5 @@
 **/.DS_Store
 **/nodemailer.config.ts
 .vscode
+package-lock.json
+package.json
```

Comment:
Is this part of the changes needed for eslint on vscode?

Reply:
When I configured my eslint, it created these 2 files that had some of my local settings so I decided to gitignore it.